src/libraries/Microsoft.Extensions.Logging/tests/Common/LoggingServiceCollectionExtensionsTest.cs

```diff
@@ -23,6 +25,42 @@ public void AddLogging_WrapsServiceCollection()
             Assert.True(callbackCalled);
         }
 
+        [Fact]
+        public void AddLogging_InjectScopeProvider()
+        {
+            var services = new ServiceCollection();
+            services.AddSingleton(typeof(IExternalScopeProvider), Mock.Of<IExternalScopeProvider>());
+
+            var callbackCalled = false;
+            var loggerBuilder = services.AddLogging(builder =>
+            {
+                callbackCalled = true;
+            });
+
+            Assert.True(callbackCalled);
+        }
+
+        [Fact]
+        public void AddLogging_TestInjectedScopeProvider()
+        {
+            var testScope = "test scope";
+            var services = new ServiceCollection();
+            var externalScopeProvider = new Mock<IExternalScopeProvider>();
+            var callbackCalled = false;
+            externalScopeProvider
+                .Setup(e => e.Push(It.IsAny<object?>()))
+                .Callback(() => callbackCalled = true);
+
+            services.AddSingleton(externalScopeProvider.Object);
+            var loggerBuilder = services.AddLogging();
+            var serviceProvider = services.BuildServiceProvider();
+            var loggerFactory = serviceProvider.GetService<ILoggerFactory>();
+            var logger = loggerFactory.CreateLogger<LoggingServiceCollectionExtensionsTest>();
+            logger.BeginScope(testScope);
+
+            Assert.True(callbackCalled);
```

Comment:
nit suggestion

```suggestion
            bool callbackCalled = false;
            var externalScopeProvider = new Mock<IExternalScopeProvider>();
            externalScopeProvider
                .Setup(e => e.Push(It.IsAny<object?>()))
                .Callback(() => callbackCalled = true);
      
            var serviceProvider = new ServiceCollection()
                .AddSingleton(externalScopeProvider.Object)
                .AddLogging()
                .BuildServiceProvider();

            var loggerFactory = serviceProvider.GetService<ILoggerFactory>();
            var logger = loggerFactory.CreateLogger<LoggingServiceCollectionExtensionsTest>();
            logger.BeginScope("test scope");

            Assert.True(callbackCalled);
```

Reply:
Thanks for the suggestion, I've applied it